single_kernel_mongo/utils/mongo_config.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2024 Canonical Ltd.
```

Comment:
Question: How did you decide which constants go in these files?

Reply:
I could see this as becoming difficult when tracking down different constants in the developer process, but I am curious what you think

Reply:
My feeling is that those constants for most of them will not be interacted a lot with except the users. The users are defined in a specific file, but the helpers constants like `ADMIN_AUTH_SOURCE` should usually not be interacted with